src/Rpc.re

```diff
@@ -29,6 +31,7 @@ type scheduler = (unit => unit) => unit;
 
 let _send = (rpc, json: Yojson.Safe.t) => {
   let str = Yojson.Safe.to_string(json);
+  Log.debugf(m => m("Sending: %s\n", str));
```

Comment:
FYI, I added 'request' logging here

Reply:
Ah, I had this before as well, but removed it because I though the acknowledgment responses were the requests! I should have looked closer.